src/app/view/tools/mouseSelectTool.js

```diff
@@ -87,7 +87,7 @@ export default class MouseSelectTool extends MouseTool {
         const target = this.hitFeature(point);
         if (target) {
             if (target.selected) {
-                const feat = Registry.currentDevice.getFeatureByID(target.featureID);
+                const feat = Registry.viewManager.getFeatureByID(target.featureID);
```

Comment:
We should throw in a `viewManagerDelegate` and ditch the `Registry`

Reply:
Should that be done in all mouse tools?

Reply:
Yeah, but we can do it incrementally as we update the different tools.